lib/src/runner/configuration.dart

```diff
@@ -50,6 +50,10 @@ class Configuration {
   bool get pauseAfterLoad => _pauseAfterLoad ?? false;
   final bool _pauseAfterLoad;
 
+  /// The path to a file to load as a base configuration before this one.
+  String get includePath => _includePath;
+  final String _includePath;
```

Comment:
No need for a private field and getter when it's final.

Reply:
Good catch, done!